zed_components/src/zed_camera/include/zed_camera_component.hpp

```diff
@@ -584,6 +585,17 @@ class ZedCamera : public rclcpp::Node
     const std::string mSrvStopSvoRecName = "stop_svo_rec";
     const std::string mSrvToggleSvoPauseName = "toggle_svo_pause";
     // <---- Services names
+
+    sl::Mat mat_left,mat_left_raw;
+    sl::Mat mat_right,mat_right_raw;
+    sl::Mat mat_left_gray,mat_left_raw_gray;
+    sl::Mat mat_right_gray,mat_right_raw_gray;
+    sl::Mat mat_depth,mat_disp,mat_conf;
+    sl::Timestamp ts_rgb=0;       // used to check RGB/Depth sync
+    sl::Timestamp ts_depth=0;     // used to check RGB/Depth sync
+    sl::Timestamp grab_ts=0;
+    rclcpp::Time timeStamp;
+
```

Comment:
What do you mean by "not POD"? I suppose it, but I want to be sure that we are speaking about he same thing

Reply:
Plain-Old Data, in this case they are "fundamental types" like size_t so they are just integers.

Reply:
Sorry, in the other comment size_t is a POD surely. Those types in contrast are classes with their contructors and destructors, maybe also virtual tables. So allocating/deallocating them each time may have a cost associated.

Reply:
OK, I guessed right. I can understand what you want to achieve. I think it's better to leave them in the function where they are used because that's the only function and `static` is OK

Reply:
Please change it back and add the `static` declaration. Then I will test the PR and merge it

Reply:
Agreed. Maybe today I already push the changes.

Reply:
Done!